calblueprint/meepanyar-node
# Wrote script and endpoint to update financial summary information

This PR:
1. Introduces an endpoint that updates a site's financial summary data given a month, year, and siteId
2. Introduces a script that calls this endpoint to update all financial summary sites. This script is going to be set to be called once per day via Heroku's cron scheduler
3. Adds a `SERVER_URL` environment variable to be filled out. The script will target the endpoint at `SERVER_URL`

### How to Review:
1. Delete the "East Asian Library" financial summary for April
2. Fill out the environment variable for `SERVER_URL` in your `.env` file. This should be pointed at your backend instance. Mee Panyar's backend instance is defaulted to run locally at `http://localhost:4000`
3. Run an instance of `meepanyar-node`
4. In another terminal, run the script via `npm run update-summaries`
5. Verify that the "East Asian Library" financial summary is created
6. Verify the values of the "East Asian Library" financial summary (Look at the Meter Readings, Payments, etc and make sure the numbers add up accurately)
7. Create a Meter Reading or Payment for an "East Asian Library" customer and rerun the script and make sure the April financial summary is appropriately updated
